docs/docs/aztec/concepts/smart_contracts/functions/function_types_macros.md

```diff
@@ -1,8 +1,29 @@
 ---
-title: Public, Private, and Unconstrained Functions
+title: Function Macros
+sidebar_position: 2
+tags: [functions, macros]
 ---
 
-This page explains the three types of functions that exist on Aztec - public, private, and unconstrained. For a deeper dive into how these functions work under the hood, check out the [Inner Workings](./inner_workings.md) page.
+This page explains three types of functions that exist on Aztec - public, private, and unconstrained; as well as all macros.
+
+# All Aztec macros
```

Comment:
```suggestion
## All Aztec macros
```
Only one `#` makes this a title heading and causes the page title to be removed from the top of the page.

![image](https://github.com/AztecProtocol/aztec-packages/assets/18372439/9cf0a2d3-4a89-4133-9d6c-0e957754fd24)